util/taskcluster.py

```diff
@@ -146,6 +146,9 @@ def main():
 
     maybe_download_tc(target_dir=args.target, tc_url=get_tc_url(args.arch, args.artifact, args.branch))
 
+    if args.artifact == "convert_graphdef_memmapped_format":
+        subprocess.check_call(['chmod', '+x', os.path.join(args.target, args.artifact)])
```

Comment:
I guess we could use `os.chmod` to have something nicer (and that works also on Windows): https://docs.python.org/dev/library/os.html#os.chmod

Reply:
Done :+1: